src/Arcus.Security.Core/Storage/Source/ConfigurationSecretProvider.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Threading.Tasks;
+using GuardNet;
+using Microsoft.Extensions.Configuration;
+
+namespace Arcus.Security.Core.Storage.Source
+{
+    /// <summary>
+    /// <see cref="ISecretProvider"/> implementation that retrieves secrets from the <see cref="IConfiguration"/>.
```

Comment:
I would emphasize dev purpose only as recommendation
```suggestion
    /// <see cref="ISecretProvider"/> implementation that retrieves secrets from the <see cref="IConfiguration"/>. It is recommended to only use this for development purposes.
```